screendoor_sdk/screendoor.py

```diff
@@ -52,7 +52,7 @@ def update_project_response(self, project_id,
                                 response_id, response_fields=None,
                                 status=None, labels=None, force_validation=False):
         """ Update project response """
-        response = {}
+        response = None
         url = self.get_url({
```

Comment:
if we use a requests.patch instead of a requests.put, will that help us avoid having to get the object every time?  would be nice to just make one call instead of two.

Reply:
We are just making one API call to make the update?

And according to the API doc, it only supports `PUT`. 😐

Reply:
Boooo! 💩